shared.nix

```diff
@@ -8,7 +8,9 @@
 
   nix.settings.builders-use-substitutes = true;
 
-  nix.settings.max-jobs = lib.mkDefault 4;
+  nix.settings.max-jobs = lib.mkDefault "auto";
+  nix.settings.keep-outputs = true; # Nice for developers
+  nix.settings.keep-derivations = true; # Idem
```

Comment:
Why do we want this on a server ?

Reply:
it's an experiment to see if it resolves errors about invalid drv store paths in the build logs